Grigore Andrei Vlad/statements/main.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{unsigned n,i,d,OK;
+scanf("%d",&n);
+printf("The prime numbers are\n");
+for(i=1;i<=n;i++)
+{OK=1;
+    for(d=2;d<=i/2;d++)
+    {
+        if(i%d==0)
+        {
+            OK=0;
+            break;
+        }
+
+        if(OK=1)
```

Comment:
Well this doesn't look right. You're actually assigning the value 1 to the variable `OK` inside that if, you're not checking to see if its value is really 1. You should I think write `if(OK==1)` instead.